extensions/hibernate-validator/runtime/src/main/java/io/quarkus/hibernate/validator/runtime/jaxrs/ResteasyReactiveViolationExceptionMapper.java

```diff
@@ -75,9 +90,15 @@ private Response buildViolationReportResponse(ConstraintViolationException cve)
      * This also what Reactive Routes uses
      */
     public static class ViolationReport {
```

Comment:
Converting the ViolationReport into a mutable object, so some serializers like JAXB can work fine with it.